Makefile

```diff
@@ -15,27 +15,37 @@ format:
 	$(CLANG_FORMAT) --style=Google -i --sort-includes $(call rwildcard,frontend/,*.java)
 	$(DART_CLI) format $(FLUTTER_APP_ROOT)
 
+test-secrets:
+	mkdir test-secrets
+	openssl aes-256-cbc -d -in encrypted/ui-maps-places-test -out test-secrets/ui-maps-places-test
```

Comment:
Please make sure to only encrypt a single payload once ever (with the same key) with raw CBC. I’m not sure if the OpenSSL tool automatically adds a random IV.